crates/algokit_transact/src/transactions/mod.rs

```diff
@@ -130,10 +132,16 @@ pub struct SignedTransaction {
     #[serde(rename = "txn")]
     pub transaction: Transaction,
 
-    /// The Ed25519 signature authorizing the transaction.
+    /// Optional Ed25519 signature authorizing the transaction.
     #[serde(rename = "sig")]
-    #[serde_as(as = "Bytes")]
-    pub signature: [u8; ALGORAND_SIGNATURE_BYTE_LENGTH],
+    #[serde_as(as = "Option<Bytes>")]
```

Comment:
I assume we switched to Option here because in the future there might be an lsig or msig?